clang-tools-extra/clangd/XRefs.cpp

```diff
@@ -2346,11 +2346,11 @@ outgoingCalls(const CallHierarchyItem &Item, const SymbolIndex *Index) {
     // Filter references to only keep function calls
```

Comment:
This comment doesn't really apply anymore.

Reply:
Reworded